ci/tasks/publish.sh

```diff
@@ -9,10 +9,11 @@ for file in $COPY_KEYS ; do
   file="${file/\%s/$VERSION}"
 
   echo "$file"
+  filename=$(${file} | grep -E -o 'bosh-stemcell.*.tgz' )
 
   # occasionally this fails for unexpected reasons; retry a few times
   for i in {1..4}; do
-    aws s3 cp "s3://$CANDIDATE_BUCKET_NAME/$file" "s3://$PUBLISHED_BUCKET_NAME/$file" \
+    aws s3 cp --content-disposition filename=${filename} --metadata-directive REPLACE "s3://$CANDIDATE_BUCKET_NAME/$file" "s3://$PUBLISHED_BUCKET_NAME/$file" \
```

Comment:
This is setting the filename to the exact name of the object, so this is essentially a no-op. I don't think we should be duplicating that information.